### DISCUSSION TITLE
Multi file writer not working for logging to both file and io::stdout

### DISCUSSION THREAD

**Discussion by hyc3z** [tokio-rs/tracing#2244] [General]
### Version

tracing = "0.1.35"
tracing-appender = "0.2.2"
tracing-subscriber = "0.3.14"

tracing v0.1.35
│   │   │   │       ├── tracing-attributes v0.1.22 (proc-macro)
│   │   │   │       └── tracing-core v0.1.28
│   │   │   └── tracing v0.1.35 (*)
│   │   ├── tracing v0.1.35 (*)
│   │   └── tracing v0.1.35 (*)
│   │   └── tracing v0.1.35 (*)
│   ├── tracing v0.1.35 (*)
├── tracing v0.1.35 (*)
├── tracing-appender v0.2.2
│   └── tracing-subscriber v0.3.14
│       ├── tracing-core v0.1.28 (*)
│       └── tracing-log v0.1.3
│           └── tracing-core v0.1.28 (*)
├── tracing-subscriber v0.3.14 (*)

### Platform

macos 12.4 monterey

### Description

In the main function:
```
let file_appender = tracing_appender::rolling::never("./logs", "log.log");

tracing_subscriber::fmt()
    .with_writer(file_appender)
    .with_ansi(false)
    .with_max_level(tracing::Level::TRACE)
    .init();
```

Above works, the log successfully outputs to log file.
However, when adding an additional writer to std::io::stdout, the log will only appear in the terminal, not in the file.
```
let file_appender = tracing_appender::rolling::never("./logs", "log.log");

tracing_subscriber::fmt()
    .with_writer(file_appender)
    .with_writer(std::io::stdout)
    .with_ansi(false)
    .with_max_level(tracing::Level::TRACE)
    .init();
```

When I follow the example here:
https://github.com/tokio-rs/tracing/blob/master/examples/examples/appender-multifile.rs

I get the following output:
```
no method named `and` found for struct `tracing_appender::rolling::RollingFileAppender` in the current scope
```

**Comment by davidbarsky** [comment]
> However, when adding an additional writer to `std::io::stdout`, the log will only appear in the terminal, not in the file.

In this example below this sentence, you're _replacing_ the `file_appender` writer with the `std::io::stdout` writer, which explains the behavior you're seeing.

> When I follow the example here:
https://github.com/tokio-rs/tracing/blob/master/examples/examples/appender-multifile.rs

Do you have `use tracing_subscriber::fmt::writer::MakeWriterExt;` at the top of your file? The `.and` combinator is defined on that extension trait.

**Reply by hyc3z** [reply]
Thanks a lot for your response! 
I tried and added
`use tracing_subscriber::fmt::writer::MakeWriterExt;`
and it is now logging to both file and stdout.

**Reply by davidbarsky** [reply]
glad to hear that it's working now!
